ethereum/spec/src/test/java/tech/pegasys/teku/spec/logic/versions/capella/BlockProcessorCapellaTest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright ConsenSys Software Inc., 2022
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.spec.logic.versions.capella;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+import java.util.Optional;
+import org.junit.jupiter.api.Test;
+import tech.pegasys.teku.bls.BLSSignatureVerifier;
+import tech.pegasys.teku.spec.Spec;
+import tech.pegasys.teku.spec.TestSpecFactory;
+import tech.pegasys.teku.spec.datastructures.blocks.SignedBeaconBlock;
+import tech.pegasys.teku.spec.datastructures.state.beaconstate.BeaconState;
+import tech.pegasys.teku.spec.logic.common.statetransition.exceptions.StateTransitionException;
+import tech.pegasys.teku.spec.logic.versions.bellatrix.BlockProcessorBellatrixTest;
+import tech.pegasys.teku.spec.util.DataStructureUtil;
+
+public class BlockProcessorCapellaTest extends BlockProcessorBellatrixTest {
+  @Override
+  protected Spec createSpec() {
+    return TestSpecFactory.createMainnetCapella();
+  }
+
+  @Test
+  void shouldIgnoreBellatrixBlock() {
```

Comment:
nit: shouldRejectBellatrixBlock? And probably rename `shouldIgnoreAltairBlock` while we're at it.

Reply:
fair